pkg/apis/flows/v1/sequence_types.go

```diff
@@ -124,10 +124,11 @@ type SequenceStatus struct {
 	// Matches the Spec.Steps array in the order.
 	ChannelStatuses []SequenceChannelStatus `json:"channelStatuses"`
 
-	// AddressStatus is the starting point to this Sequence. Sending to this
+	// Address is the starting point to this Sequence. Sending to this
```

Comment:
Just to make sure I understand correctly. This is equivalent because we inlined AddressStatus before and instead of inlining you changed it to be explicit? Is there a reason why this is not a pointer like what AddressStatus has?

https://github.com/knative/pkg/blob/main/apis/duck/v1/addressable_types.go#L64

Reply:
Yeah, should've been clearer. I saw there being inconsistencies also. But just wanted to make sure that my understanding is that there are no changes to wire format. We should probably vet these to make sure they are consistent, _and_ make sure that we comment the duck properly since it says there to embed it like this (well, that's how I read it still).